src/lib/containers/widgets/facet-nav/FacetNav.tsx

```diff
@@ -40,33 +40,31 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
   loadingScreen,
   isLoading,
   executeQueryRequest,
+  token,
   asyncJobStatus,
-  facetsToPlot,
+  show,
 }: FacetNavProps): JSX.Element => {
   const [facetUiStateArray, setFacetUiStateArray] = useState<UiFacetState[]>([])
   const [expandedFacets, setExpandedFacets] = useState<ExpandedFacet[]>([])
   const [isFirstTime, setIsFirstTime] = useState(true)
 
   const request = getLastQueryRequest!()
-  const getFacets = (data: QueryResultBundle | undefined): FacetColumnResult[] =>  {
+  const getFacets = (
+    data: QueryResultBundle | undefined,
+  ): FacetColumnResult[] => {
     const result = data?.facets?.filter(
-      item =>
-        item.facetType === 'enumeration' &&
-        (!facetsToPlot?.length || facetsToPlot.indexOf(item.columnName) > -1)
+      item => item.facetType === 'enumeration',
```

Comment:
e we are not supporting the ability to select the facets to display in vis?

Reply:
Whoops, let me add that back, took it out for an unrelated reason but forgot to add it back.